src/ucp/tag/rndv.c

```diff
@@ -853,9 +857,14 @@ static ucs_status_t ucp_rndv_pipeline(ucp_request_t *sreq, ucp_rndv_rtr_hdr_t *r
 
     ucs_assert(!UCP_MEM_IS_HOST(sreq->send.mem_type));
 
+    /* check if lane supports GET Zcopy operation on a iface */
+    iface_attr = ucp_ep_get_iface_attr(sreq->send.ep, sreq->send.lane);
```

Comment:
can we do this check before calling ucp_rndv_pipeline()? also for consistency need to check ep_config->max_get_zcopy rather than iface cap flag

Reply:
done